project-p4-chatbot-master/code/script.js

```diff
@@ -0,0 +1,293 @@
+  <button id="jobsurferBtn">Ich möchte gerne Jobsurfer werden und Erfahrungen in neuen Berufsfelder sammeln.</button>
+  <button id="companyBtn">Mein Unternehmen möchte Quereinsteigern Joberfahrungen ermöglichen.</button>
+  <button id="justBtn">Ich möchte mich nur mal umschauen.</button>
+  `
+  document.getElementById('jobsurferBtn').addEventListener('click', handleJobsurferInput);  
+  document.getElementById('companyBtn').addEventListener('click', handleCompanyInput);
+  document.getElementById('justBtn').addEventListener('click', handleJustInput);
+}
+
+const handleTiredInput = (motivation) => {
+  console.log("tired")
+  showMessage("Müde und ausgelaugt", 'user')
+
+  setTimeout(() => showTiredOptions(), 1000)
+}
+
+const showTiredOptions = (interest) => {
+
+  showMessage(`Müde und ausgelaugt? Das kenn ich. Na dann geb ich mein Bestes, dass du dich besser fühlst! 😄
+  Warum interessierst du dich denn für jobsurfing.ch?`, 'bot')
+
+  inputWrapper.innerHTML = `
+  <button id="jobsurferBtn">Ich möchte gerne Jobsurfer werden und Erfahrungen in neuen Berufsfelder sammeln.</button>
+  <button id="companyBtn">Mein Unternehmen möchte Quereinsteigern Joberfahrungen ermöglichen.</button>
+  <button id="justBtn">Ich möchte mich nur mal umschauen.</button>
+  `
+  document.getElementById('jobsurferBtn').addEventListener('click', handleJobsurferInput);  
+  document.getElementById('companyBtn').addEventListener('click', handleCompanyInput);
+  document.getElementById('justBtn').addEventListener('click', handleJustInput);
+
+}
+
+const handleOkayInput = (motivation) => {
+  console.log("okay")
+  showMessage("Ganz okay", 'user')
+
+  setTimeout(() => showOkayOptions(), 1000)
+}
+
+const showOkayOptions = (interest) => {  
+  showMessage(`Ganz okay? Na da geht doch noch was. 😉 Ich gebe mein Bestes, deine Stimmung noch etwas anzuheben!
+  Warum interessierst du dich denn für jobsurfing.ch?`, 'bot')
+
+  inputWrapper.innerHTML = `
+  <button id="jobsurferBtn">Ich möchte gerne Jobsurfer werden und Erfahrungen in neuen Berufsfeldern sammeln.</button>
+  <button id="companyBtn">Mein Unternehmen möchte Quereinsteigern Joberfahrungen ermöglichen.</button>
+  <button id="justBtn">Ich möchte mich nur mal umschauen.</button>
+  `
+  document.getElementById('jobsurferBtn').addEventListener('click', handleJobsurferInput);  
+  document.getElementById('companyBtn').addEventListener('click', handleCompanyInput);
+  document.getElementById('justBtn').addEventListener('click', handleJustInput);
+}
+
+const handleJobsurferInput = (reason) => {
+  console.log("jobsurfer")
+  showMessage("Ich möchte gerne Jobsurfer werden und Erfahrungen in neuen Berufsfelder sammeln.", 'user')
+
+  setTimeout(() => showJobsurferOptions(), 1000)
+}
+
+const showJobsurferOptions = (reason) => {
+  showMessage(`👏So toll, dass du neue berufliche Erfahrungen sammeln willst. Welcher Grund ist ausschlaggebend dafür?`, 'bot')
+
+  inputWrapper.innerHTML = `
+    <select id="select"> 
```

Comment:
Maybe styling the `select`s to look a bit more like the rest of the chat would be nice? :)